tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+requires =
+	tox>=4
+env_list = lint, integration
+
+[testenv]
+setenv =
+	version=3.3.2
+	repo=https://github.com/canonical/kafka-k8s-operator.git
+	base-name=docker.io/charmed/charmed-kafka
+
+[testenv:lint]
+description = run linters
+skip_install = true
+deps =
+	yamllint
+commands = 
+	yamllint --no-warnings rockcraft.yaml .github
+
+[testenv:integration]
+description = run operator integration tests
+skip_install = true
+allowlist_externals =
+	tox
+	microk8s
+	bash
+commands = 
+	bash -ec 'if ! [ -f charmed-kafka_{env:version}_amd64.rock ]; then rockcraft pack; fi' {posargs}
+	microk8s ctr image import charmed-kafka_{env:version}_amd64.rock --base-name {env:base-name}
```

Comment:
we might need to tune {env:version} here, as {env:base-name} == `docker.io/charmed/kafka` (`latest` tag) but you want to use/pull `docker.io/charmed/kafka:{env:version}`. Let's merge and test it and tune later.

Reply:
The version will be extracted directly from the rock image. I tested it and it works properly